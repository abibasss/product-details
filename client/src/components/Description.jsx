import React from 'react';

const Description = (props) => {
  return (
    <div className = "container">
      <div className="row">
        <div className="col-sm shoe-desc">
          <h2 className="detail-name">{props.shoe.name}</h2>
          <h5 className = "short-desc">{props.shoe.short_desc}</h5>
          <p className= "para">{props.shoe.long_desc}</p>
        </div>
        <div className="col-sm shoe-img">
          <img src="https://assets.adidas.com/images/w_840,h_840,f_auto,q_auto:sensitive,fl_lossy/29ffefeadfb448bfae6ba93b007c3f38_9366/POD-S3_1_Shoes_Black_G54741_04_standard.jpg" className="description_img___3nrC6" alt="POD-S3.1 Shoes" width="430" height="430"></img>
        </div>
      </div>
    </div>
  )
}

export default Description;
