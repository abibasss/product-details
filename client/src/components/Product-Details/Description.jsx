import React from 'react';

const Description = (props) => {
  return (
    <div className = "container">
      <div className="row">
        <div className="col-sm shoe-desc">
          <h2 className="detail-name">{props.shoe.name}</h2>
          <h5 className = "short-desc">{props.shoe.short_desc}</h5>
          <p className= "para">{props.shoe.long_desc}</p>
        </div>
        <div className="col-sm shoe-img">
          <img src="https://assets.adidas.com/images/w_840,h_840,f_auto,q_auto:sensitive,fl_lossy/29ffefeadfb448bfae6ba93b007c3f38_9366/POD-S3_1_Shoes_Black_G54741_04_standard.jpg" width="420" height="420"></img>
        </div>
      </div>
    </div>
  )
}

// const Description = (props) => {
//   return (
//     <div className = "container">
//       <div className="row">
//         <div className="col-sm shoe-desc">
//           <h2 className="detail-name">{props.shoe.name}</h2>
//           <h5 className = "short-desc">{props.shoe.short_desc}</h5>
//           <p className= "para">{props.shoe.long_desc}</p>
//         </div>
//         <div className="col-sm shoe-img">
//           <img src={props.shoe.img_url} width="420" height="420"></img>
//         </div>
//       </div>
//     </div>
//   )
// }

export default Description;
