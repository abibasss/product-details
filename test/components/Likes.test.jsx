import React from 'react';
import { shallow, mount, render } from 'enzyme';

import Likes from '../../client/src/components/Likes';
import Description from '../../client/src/components/Description';

describe('Likes component', () => {
  const sample = [{
      "id": 1,
      "name": "SOLARBOOST",
      "img_url": "https://s3-us-west-1.amazonaws.com/adidas-shoe/14.jpg",
      "type": "Performance",
      "price": 110,
      "short_desc": 'This is a short description for solarboost',
      "long_desc": 'This is a long description of solarboost'
    },
    {
      "id": 2,
      "name": "ANDALAX",
      "img_url": "https://s3-us-west-1.amazonaws.com/adidas-shoe/13.jpg",
      "type": "Comfort",
      "price": 115,
      "short_desc": 'This is a short description for andalax',
      "long_desc": 'This is a long description of andalax'
    },
    {
      "id": 3,
      "name": "SOLAR GLIDE",
      "img_url": "https://s3-us-west-1.amazonaws.com/adidas-shoe/12.jpg",
      "type": "Originals",
      "price": 130
    },
    {
      "id": 4,
      "name": "ULTRABOOST",
      "img_url": "https://s3-us-west-1.amazonaws.com/adidas-shoe/ultraboost-black.jpg",
      "type": "Performance",
      "price": 95,
    },
    {
      "id": 5,
      "name": "SOLAR GLIDE ST",
      "img_url": "https://s3-us-west-1.amazonaws.com/adidas-shoe/11.jpg",
      "type": "Performance",
      "price": 140,
    },
    {
      "id": 6,
      "name": "SOLARBOOST",
      "img_url": "https://s3-us-west-1.amazonaws.com/adidas-shoe/14.jpg",
      "type": "Performance",
      "price": 110
    },
    {
      "id": 7,
      "name": "ANDALAX",
      "img_url": "https://s3-us-west-1.amazonaws.com/adidas-shoe/13.jpg",
      "type": "Comfort",
      "price": 115
    },
    {
      "id": 8,
      "name": "SOLAR GLIDE",
      "img_url": "https://s3-us-west-1.amazonaws.com/adidas-shoe/12.jpg",
      "type": "Originals",
      "price": 130
    },
    {
      "id": 9,
      "name": "ULTRABOOST",
      "img_url": "https://s3-us-west-1.amazonaws.com/adidas-shoe/ultraboost-black.jpg",
      "type": "Performance",
      "price": 95,
    },
    {
      "id": 10,
      "name": "SOLAR GLIDE ST",
      "img_url": "https://s3-us-west-1.amazonaws.com/adidas-shoe/11.jpg",
      "type": "Performance",
      "price": 140,
    },
    {
      "id": 11,
      "name": "SOLARBOOST",
      "img_url": "https://s3-us-west-1.amazonaws.com/adidas-shoe/14.jpg",
      "type": "Performance",
      "price": 110
    },
    {
      "id": 12,
      "name": "ANDALAX",
      "img_url": "https://s3-us-west-1.amazonaws.com/adidas-shoe/13.jpg",
      "type": "Comfort",
      "price": 115
    },
    {
      "id": 13,
      "name": "SOLAR GLIDE",
      "img_url": "https://s3-us-west-1.amazonaws.com/adidas-shoe/12.jpg",
      "type": "Originals",
      "price": 130
    },
    {
      "id": 14,
      "name": "ULTRABOOST",
      "img_url": "https://s3-us-west-1.amazonaws.com/adidas-shoe/ultraboost-black.jpg",
      "type": "Performance",
      "price": 95,
    },
    {
      "id": 15,
      "name": "SOLAR GLIDE ST",
      "img_url": "https://s3-us-west-1.amazonaws.com/adidas-shoe/11.jpg",
      "type": "Performance",
      "price": 140,
    },
  ];

  const component = shallow(<Likes shoes={sample} />);


  it('should display like snapshot', () => {
    expect(component).toMatchSnapshot();
  })

  describe('Image Carousel', () => {
    it('should display carousel of shoes properly', () => {
      expect(component.find('div#carouselLikes')).toHaveLength(1);
    });

    it('should start with first slide', () => {
      expect(component.find('li.active')).toHaveLength(1);
      expect(component.find('li').at(0)).toHaveClassName('active');
    });

    it('should display 4 pages of image cards', () => {
      expect(component.find('div.carousel-item')).toHaveLength(4);
    })

    it('should have at least have 2 image cards', () => {
      expect(component.find('div.card').length).toBeGreaterThan(2);
    })

    it('should display name, type, and price on each card', () => {
      expect(component.find('span#like-name').at(0).text()).toBe('SOLARBOOST');
      expect(component.find('span#like-type').at(0).text()).toBe('Performance');
      expect(component.find('span#like-price').at(0).text()).toBe('$110');
    })

    it('should render images in order', () => {
      expect(component.find('span#like-name').at(0).text()).toBe('SOLARBOOST');
      expect(component.find('span#like-name').at(1).text()).toBe('ANDALAX');
      expect(component.find('span#like-price').at(2).text()).toBe('$130');
    })
  })


})
