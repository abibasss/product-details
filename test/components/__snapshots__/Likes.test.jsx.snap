// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Likes component should display like snapshot 1`] = `
<div
  className="like"
>
  <div
    className="carousel slide"
    data-interval="false"
    data-wrap="false"
    id="carouselLikes"
  >
    <ol
      className="carousel-indicators"
    >
      <li
        className="active"
        data-slide-to="0"
        data-target="#carouselLikes"
        onClick={[Function]}
      />
      <li
        data-slide-to="1"
        data-target="#carouselLikes"
        onClick={[Function]}
      />
      <li
        data-slide-to="2"
        data-target="#carouselLikes"
        onClick={[Function]}
      />
      <li
        data-slide-to="3"
        data-target="#carouselLikes"
        onClick={[Function]}
      />
    </ol>
    <div
      className="carousel-inner"
    >
      <div
        className="carousel-item active"
      >
        <div
          id="likes"
        >
          <div
            className="card"
            key="1"
            onClick={[Function]}
          >
            <img
              className="card-img-top likes d-block"
              src="https://www.adidas.com/dis/dw/image/v2/aaqx_prd/on/demandware.static/-/Sites-adidas-products/en_US/dwa800131d/zoom/CM8322_00_plp_standard.jpg?sw=276&sh=276&sm=fit&hei=276&wid=276&strip=false"
            />
            <div
              className="card-body"
            >
              <span
                id="like-type"
              >
                Performance
              </span>
              <br />
              <span
                id="like-name"
              >
                SOLARBOOST
              </span>
              <br />
              <span
                id="like-price"
              >
                $
                110
              </span>
              <br />
              <div
                className="rating-review"
              >
                <RatingAPILayer
                  direction="ltr"
                  emptySymbol="far fa-star"
                  fractions={5}
                  fullSymbol="fa fa-star"
                  initialRating={NaN}
                  onChange={[Function]}
                  onClick={[Function]}
                  onHover={[Function]}
                  placeholderSymbol={
                    Object {
                      "backgroundColor": "red",
                      "border": "5px double white",
                      "borderRadius": "50%",
                      "display": "inline-block",
                      "height": 30,
                      "width": 30,
                    }
                  }
                  quiet={false}
                  readonly={true}
                  start={0}
                  step={1}
                  stop={5}
                />
                <span
                  className="rating"
                />
              </div>
            </div>
          </div>
          <div
            className="card"
            key="2"
            onClick={[Function]}
          >
            <img
              className="card-img-top likes d-block"
              src="https://www.adidas.com/dis/dw/image/v2/aaqx_prd/on/demandware.static/-/Sites-adidas-products/en_US/dwa800131d/zoom/CM8322_00_plp_standard.jpg?sw=276&sh=276&sm=fit&hei=276&wid=276&strip=false"
            />
            <div
              className="card-body"
            >
              <span
                id="like-type"
              >
                Comfort
              </span>
              <br />
              <span
                id="like-name"
              >
                ANDALAX
              </span>
              <br />
              <span
                id="like-price"
              >
                $
                115
              </span>
              <br />
              <div
                className="rating-review"
              >
                <RatingAPILayer
                  direction="ltr"
                  emptySymbol="far fa-star"
                  fractions={5}
                  fullSymbol="fa fa-star"
                  initialRating={NaN}
                  onChange={[Function]}
                  onClick={[Function]}
                  onHover={[Function]}
                  placeholderSymbol={
                    Object {
                      "backgroundColor": "red",
                      "border": "5px double white",
                      "borderRadius": "50%",
                      "display": "inline-block",
                      "height": 30,
                      "width": 30,
                    }
                  }
                  quiet={false}
                  readonly={true}
                  start={0}
                  step={1}
                  stop={5}
                />
                <span
                  className="rating"
                />
              </div>
            </div>
          </div>
          <div
            className="card"
            key="3"
            onClick={[Function]}
          >
            <img
              className="card-img-top likes d-block"
              src="https://www.adidas.com/dis/dw/image/v2/aaqx_prd/on/demandware.static/-/Sites-adidas-products/en_US/dwa800131d/zoom/CM8322_00_plp_standard.jpg?sw=276&sh=276&sm=fit&hei=276&wid=276&strip=false"
            />
            <div
              className="card-body"
            >
              <span
                id="like-type"
              >
                Originals
              </span>
              <br />
              <span
                id="like-name"
              >
                SOLAR GLIDE
              </span>
              <br />
              <span
                id="like-price"
              >
                $
                130
              </span>
              <br />
              <div
                className="rating-review"
              >
                <RatingAPILayer
                  direction="ltr"
                  emptySymbol="far fa-star"
                  fractions={5}
                  fullSymbol="fa fa-star"
                  initialRating={NaN}
                  onChange={[Function]}
                  onClick={[Function]}
                  onHover={[Function]}
                  placeholderSymbol={
                    Object {
                      "backgroundColor": "red",
                      "border": "5px double white",
                      "borderRadius": "50%",
                      "display": "inline-block",
                      "height": 30,
                      "width": 30,
                    }
                  }
                  quiet={false}
                  readonly={true}
                  start={0}
                  step={1}
                  stop={5}
                />
                <span
                  className="rating"
                />
              </div>
            </div>
          </div>
          <div
            className="card"
            key="4"
            onClick={[Function]}
          >
            <img
              className="card-img-top likes d-block"
              src="https://www.adidas.com/dis/dw/image/v2/aaqx_prd/on/demandware.static/-/Sites-adidas-products/en_US/dwa800131d/zoom/CM8322_00_plp_standard.jpg?sw=276&sh=276&sm=fit&hei=276&wid=276&strip=false"
            />
            <div
              className="card-body"
            >
              <span
                id="like-type"
              >
                Performance
              </span>
              <br />
              <span
                id="like-name"
              >
                ULTRABOOST
              </span>
              <br />
              <span
                id="like-price"
              >
                $
                95
              </span>
              <br />
              <div
                className="rating-review"
              >
                <RatingAPILayer
                  direction="ltr"
                  emptySymbol="far fa-star"
                  fractions={5}
                  fullSymbol="fa fa-star"
                  initialRating={NaN}
                  onChange={[Function]}
                  onClick={[Function]}
                  onHover={[Function]}
                  placeholderSymbol={
                    Object {
                      "backgroundColor": "red",
                      "border": "5px double white",
                      "borderRadius": "50%",
                      "display": "inline-block",
                      "height": 30,
                      "width": 30,
                    }
                  }
                  quiet={false}
                  readonly={true}
                  start={0}
                  step={1}
                  stop={5}
                />
                <span
                  className="rating"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="carousel-item"
      >
        <div
          id="likes"
        >
          <div
            className="card"
            key="5"
            onClick={[Function]}
          >
            <img
              className="card-img-top likes d-block"
              src="https://www.adidas.com/dis/dw/image/v2/aaqx_prd/on/demandware.static/-/Sites-adidas-products/en_US/dwa800131d/zoom/CM8322_00_plp_standard.jpg?sw=276&sh=276&sm=fit&hei=276&wid=276&strip=false"
            />
            <div
              className="card-body"
            >
              <span
                id="like-type"
              >
                Performance
              </span>
              <br />
              <span
                id="like-name"
              >
                SOLAR GLIDE ST
              </span>
              <br />
              <span
                id="like-price"
              >
                $
                140
              </span>
              <br />
              <div
                className="rating-review"
              >
                <RatingAPILayer
                  direction="ltr"
                  emptySymbol="far fa-star"
                  fractions={5}
                  fullSymbol="fa fa-star"
                  initialRating={NaN}
                  onChange={[Function]}
                  onClick={[Function]}
                  onHover={[Function]}
                  placeholderSymbol={
                    Object {
                      "backgroundColor": "red",
                      "border": "5px double white",
                      "borderRadius": "50%",
                      "display": "inline-block",
                      "height": 30,
                      "width": 30,
                    }
                  }
                  quiet={false}
                  readonly={true}
                  start={0}
                  step={1}
                  stop={5}
                />
                <span
                  className="rating"
                />
              </div>
            </div>
          </div>
          <div
            className="card"
            key="6"
            onClick={[Function]}
          >
            <img
              className="card-img-top likes d-block"
              src="https://www.adidas.com/dis/dw/image/v2/aaqx_prd/on/demandware.static/-/Sites-adidas-products/en_US/dwa800131d/zoom/CM8322_00_plp_standard.jpg?sw=276&sh=276&sm=fit&hei=276&wid=276&strip=false"
            />
            <div
              className="card-body"
            >
              <span
                id="like-type"
              >
                Performance
              </span>
              <br />
              <span
                id="like-name"
              >
                SOLARBOOST
              </span>
              <br />
              <span
                id="like-price"
              >
                $
                110
              </span>
              <br />
              <div
                className="rating-review"
              >
                <RatingAPILayer
                  direction="ltr"
                  emptySymbol="far fa-star"
                  fractions={5}
                  fullSymbol="fa fa-star"
                  initialRating={NaN}
                  onChange={[Function]}
                  onClick={[Function]}
                  onHover={[Function]}
                  placeholderSymbol={
                    Object {
                      "backgroundColor": "red",
                      "border": "5px double white",
                      "borderRadius": "50%",
                      "display": "inline-block",
                      "height": 30,
                      "width": 30,
                    }
                  }
                  quiet={false}
                  readonly={true}
                  start={0}
                  step={1}
                  stop={5}
                />
                <span
                  className="rating"
                />
              </div>
            </div>
          </div>
          <div
            className="card"
            key="7"
            onClick={[Function]}
          >
            <img
              className="card-img-top likes d-block"
              src="https://www.adidas.com/dis/dw/image/v2/aaqx_prd/on/demandware.static/-/Sites-adidas-products/en_US/dwa800131d/zoom/CM8322_00_plp_standard.jpg?sw=276&sh=276&sm=fit&hei=276&wid=276&strip=false"
            />
            <div
              className="card-body"
            >
              <span
                id="like-type"
              >
                Comfort
              </span>
              <br />
              <span
                id="like-name"
              >
                ANDALAX
              </span>
              <br />
              <span
                id="like-price"
              >
                $
                115
              </span>
              <br />
              <div
                className="rating-review"
              >
                <RatingAPILayer
                  direction="ltr"
                  emptySymbol="far fa-star"
                  fractions={5}
                  fullSymbol="fa fa-star"
                  initialRating={NaN}
                  onChange={[Function]}
                  onClick={[Function]}
                  onHover={[Function]}
                  placeholderSymbol={
                    Object {
                      "backgroundColor": "red",
                      "border": "5px double white",
                      "borderRadius": "50%",
                      "display": "inline-block",
                      "height": 30,
                      "width": 30,
                    }
                  }
                  quiet={false}
                  readonly={true}
                  start={0}
                  step={1}
                  stop={5}
                />
                <span
                  className="rating"
                />
              </div>
            </div>
          </div>
          <div
            className="card"
            key="8"
            onClick={[Function]}
          >
            <img
              className="card-img-top likes d-block"
              src="https://www.adidas.com/dis/dw/image/v2/aaqx_prd/on/demandware.static/-/Sites-adidas-products/en_US/dwa800131d/zoom/CM8322_00_plp_standard.jpg?sw=276&sh=276&sm=fit&hei=276&wid=276&strip=false"
            />
            <div
              className="card-body"
            >
              <span
                id="like-type"
              >
                Originals
              </span>
              <br />
              <span
                id="like-name"
              >
                SOLAR GLIDE
              </span>
              <br />
              <span
                id="like-price"
              >
                $
                130
              </span>
              <br />
              <div
                className="rating-review"
              >
                <RatingAPILayer
                  direction="ltr"
                  emptySymbol="far fa-star"
                  fractions={5}
                  fullSymbol="fa fa-star"
                  initialRating={NaN}
                  onChange={[Function]}
                  onClick={[Function]}
                  onHover={[Function]}
                  placeholderSymbol={
                    Object {
                      "backgroundColor": "red",
                      "border": "5px double white",
                      "borderRadius": "50%",
                      "display": "inline-block",
                      "height": 30,
                      "width": 30,
                    }
                  }
                  quiet={false}
                  readonly={true}
                  start={0}
                  step={1}
                  stop={5}
                />
                <span
                  className="rating"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="carousel-item"
      >
        <div
          id="likes"
        >
          <div
            className="card"
            key="9"
            onClick={[Function]}
          >
            <img
              className="card-img-top likes d-block"
              src="https://www.adidas.com/dis/dw/image/v2/aaqx_prd/on/demandware.static/-/Sites-adidas-products/en_US/dwa800131d/zoom/CM8322_00_plp_standard.jpg?sw=276&sh=276&sm=fit&hei=276&wid=276&strip=false"
            />
            <div
              className="card-body"
            >
              <span
                id="like-type"
              >
                Performance
              </span>
              <br />
              <span
                id="like-name"
              >
                ULTRABOOST
              </span>
              <br />
              <span
                id="like-price"
              >
                $
                95
              </span>
              <br />
              <div
                className="rating-review"
              >
                <RatingAPILayer
                  direction="ltr"
                  emptySymbol="far fa-star"
                  fractions={5}
                  fullSymbol="fa fa-star"
                  initialRating={NaN}
                  onChange={[Function]}
                  onClick={[Function]}
                  onHover={[Function]}
                  placeholderSymbol={
                    Object {
                      "backgroundColor": "red",
                      "border": "5px double white",
                      "borderRadius": "50%",
                      "display": "inline-block",
                      "height": 30,
                      "width": 30,
                    }
                  }
                  quiet={false}
                  readonly={true}
                  start={0}
                  step={1}
                  stop={5}
                />
                <span
                  className="rating"
                />
              </div>
            </div>
          </div>
          <div
            className="card"
            key="10"
            onClick={[Function]}
          >
            <img
              className="card-img-top likes d-block"
              src="https://www.adidas.com/dis/dw/image/v2/aaqx_prd/on/demandware.static/-/Sites-adidas-products/en_US/dwa800131d/zoom/CM8322_00_plp_standard.jpg?sw=276&sh=276&sm=fit&hei=276&wid=276&strip=false"
            />
            <div
              className="card-body"
            >
              <span
                id="like-type"
              >
                Performance
              </span>
              <br />
              <span
                id="like-name"
              >
                SOLAR GLIDE ST
              </span>
              <br />
              <span
                id="like-price"
              >
                $
                140
              </span>
              <br />
              <div
                className="rating-review"
              >
                <RatingAPILayer
                  direction="ltr"
                  emptySymbol="far fa-star"
                  fractions={5}
                  fullSymbol="fa fa-star"
                  initialRating={NaN}
                  onChange={[Function]}
                  onClick={[Function]}
                  onHover={[Function]}
                  placeholderSymbol={
                    Object {
                      "backgroundColor": "red",
                      "border": "5px double white",
                      "borderRadius": "50%",
                      "display": "inline-block",
                      "height": 30,
                      "width": 30,
                    }
                  }
                  quiet={false}
                  readonly={true}
                  start={0}
                  step={1}
                  stop={5}
                />
                <span
                  className="rating"
                />
              </div>
            </div>
          </div>
          <div
            className="card"
            key="11"
            onClick={[Function]}
          >
            <img
              className="card-img-top likes d-block"
              src="https://www.adidas.com/dis/dw/image/v2/aaqx_prd/on/demandware.static/-/Sites-adidas-products/en_US/dwa800131d/zoom/CM8322_00_plp_standard.jpg?sw=276&sh=276&sm=fit&hei=276&wid=276&strip=false"
            />
            <div
              className="card-body"
            >
              <span
                id="like-type"
              >
                Performance
              </span>
              <br />
              <span
                id="like-name"
              >
                SOLARBOOST
              </span>
              <br />
              <span
                id="like-price"
              >
                $
                110
              </span>
              <br />
              <div
                className="rating-review"
              >
                <RatingAPILayer
                  direction="ltr"
                  emptySymbol="far fa-star"
                  fractions={5}
                  fullSymbol="fa fa-star"
                  initialRating={NaN}
                  onChange={[Function]}
                  onClick={[Function]}
                  onHover={[Function]}
                  placeholderSymbol={
                    Object {
                      "backgroundColor": "red",
                      "border": "5px double white",
                      "borderRadius": "50%",
                      "display": "inline-block",
                      "height": 30,
                      "width": 30,
                    }
                  }
                  quiet={false}
                  readonly={true}
                  start={0}
                  step={1}
                  stop={5}
                />
                <span
                  className="rating"
                />
              </div>
            </div>
          </div>
          <div
            className="card"
            key="12"
            onClick={[Function]}
          >
            <img
              className="card-img-top likes d-block"
              src="https://www.adidas.com/dis/dw/image/v2/aaqx_prd/on/demandware.static/-/Sites-adidas-products/en_US/dwa800131d/zoom/CM8322_00_plp_standard.jpg?sw=276&sh=276&sm=fit&hei=276&wid=276&strip=false"
            />
            <div
              className="card-body"
            >
              <span
                id="like-type"
              >
                Comfort
              </span>
              <br />
              <span
                id="like-name"
              >
                ANDALAX
              </span>
              <br />
              <span
                id="like-price"
              >
                $
                115
              </span>
              <br />
              <div
                className="rating-review"
              >
                <RatingAPILayer
                  direction="ltr"
                  emptySymbol="far fa-star"
                  fractions={5}
                  fullSymbol="fa fa-star"
                  initialRating={NaN}
                  onChange={[Function]}
                  onClick={[Function]}
                  onHover={[Function]}
                  placeholderSymbol={
                    Object {
                      "backgroundColor": "red",
                      "border": "5px double white",
                      "borderRadius": "50%",
                      "display": "inline-block",
                      "height": 30,
                      "width": 30,
                    }
                  }
                  quiet={false}
                  readonly={true}
                  start={0}
                  step={1}
                  stop={5}
                />
                <span
                  className="rating"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="carousel-item"
      >
        <div
          id="likes"
        >
          <div
            className="card"
            key="13"
            onClick={[Function]}
          >
            <img
              className="card-img-top likes d-block"
              src="https://www.adidas.com/dis/dw/image/v2/aaqx_prd/on/demandware.static/-/Sites-adidas-products/en_US/dwa800131d/zoom/CM8322_00_plp_standard.jpg?sw=276&sh=276&sm=fit&hei=276&wid=276&strip=false"
            />
            <div
              className="card-body"
            >
              <span
                id="like-type"
              >
                Originals
              </span>
              <br />
              <span
                id="like-name"
              >
                SOLAR GLIDE
              </span>
              <br />
              <span
                id="like-price"
              >
                $
                130
              </span>
              <br />
              <div
                className="rating-review"
              >
                <RatingAPILayer
                  direction="ltr"
                  emptySymbol="far fa-star"
                  fractions={5}
                  fullSymbol="fa fa-star"
                  initialRating={NaN}
                  onChange={[Function]}
                  onClick={[Function]}
                  onHover={[Function]}
                  placeholderSymbol={
                    Object {
                      "backgroundColor": "red",
                      "border": "5px double white",
                      "borderRadius": "50%",
                      "display": "inline-block",
                      "height": 30,
                      "width": 30,
                    }
                  }
                  quiet={false}
                  readonly={true}
                  start={0}
                  step={1}
                  stop={5}
                />
                <span
                  className="rating"
                />
              </div>
            </div>
          </div>
          <div
            className="card"
            key="14"
            onClick={[Function]}
          >
            <img
              className="card-img-top likes d-block"
              src="https://www.adidas.com/dis/dw/image/v2/aaqx_prd/on/demandware.static/-/Sites-adidas-products/en_US/dwa800131d/zoom/CM8322_00_plp_standard.jpg?sw=276&sh=276&sm=fit&hei=276&wid=276&strip=false"
            />
            <div
              className="card-body"
            >
              <span
                id="like-type"
              >
                Performance
              </span>
              <br />
              <span
                id="like-name"
              >
                ULTRABOOST
              </span>
              <br />
              <span
                id="like-price"
              >
                $
                95
              </span>
              <br />
              <div
                className="rating-review"
              >
                <RatingAPILayer
                  direction="ltr"
                  emptySymbol="far fa-star"
                  fractions={5}
                  fullSymbol="fa fa-star"
                  initialRating={NaN}
                  onChange={[Function]}
                  onClick={[Function]}
                  onHover={[Function]}
                  placeholderSymbol={
                    Object {
                      "backgroundColor": "red",
                      "border": "5px double white",
                      "borderRadius": "50%",
                      "display": "inline-block",
                      "height": 30,
                      "width": 30,
                    }
                  }
                  quiet={false}
                  readonly={true}
                  start={0}
                  step={1}
                  stop={5}
                />
                <span
                  className="rating"
                />
              </div>
            </div>
          </div>
          <div
            className="card"
            key="15"
            onClick={[Function]}
          >
            <img
              className="card-img-top likes d-block"
              src="https://www.adidas.com/dis/dw/image/v2/aaqx_prd/on/demandware.static/-/Sites-adidas-products/en_US/dwa800131d/zoom/CM8322_00_plp_standard.jpg?sw=276&sh=276&sm=fit&hei=276&wid=276&strip=false"
            />
            <div
              className="card-body"
            >
              <span
                id="like-type"
              >
                Performance
              </span>
              <br />
              <span
                id="like-name"
              >
                SOLAR GLIDE ST
              </span>
              <br />
              <span
                id="like-price"
              >
                $
                140
              </span>
              <br />
              <div
                className="rating-review"
              >
                <RatingAPILayer
                  direction="ltr"
                  emptySymbol="far fa-star"
                  fractions={5}
                  fullSymbol="fa fa-star"
                  initialRating={NaN}
                  onChange={[Function]}
                  onClick={[Function]}
                  onHover={[Function]}
                  placeholderSymbol={
                    Object {
                      "backgroundColor": "red",
                      "border": "5px double white",
                      "borderRadius": "50%",
                      "display": "inline-block",
                      "height": 30,
                      "width": 30,
                    }
                  }
                  quiet={false}
                  readonly={true}
                  start={0}
                  step={1}
                  stop={5}
                />
                <span
                  className="rating"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="control-icons"
    >
      <div
        className="controls"
      >
        <div
          className="row control-row"
        >
          <a
            className="carousel-control-prev hidden"
            data-slide="prev"
            href="#carouselLikes"
            onClick={[Function]}
            role="button"
          >
            <span
              aria-hidden="true"
              className="carousel-control-prev-icon"
            />
          </a>
        </div>
        <div
          className="row control-row next-pos"
        >
          <a
            className="carousel-control-next"
            data-slide="next"
            href="#carouselLikes"
            onClick={[Function]}
            role="button"
          >
            <span
              aria-hidden="true"
              className="carousel-control-next-icon"
            />
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
